src/bin/cli.ts

```diff
@@ -112,20 +112,23 @@ const run = async function () {
     process.exit(1)
   }
 
-  const hasErrors = batches.some((batch) => batch.errors.length > 0 || batch.contentTransformErrors.length > 0)
+  const hasTransformErrors = batches.some((batch) => batch.errors.length > 0 || batch.contentTransformErrors.length > 0)
```

Comment:
If you want to really check for `hasTransformErrors` only, we should split this up into two arrays, each filled by one predicate only

Reply:
ah, I wanted to make the difference to the server errors below clearer but overlooked that this is actually different types of errors. reversing this one.